JamGame/Assets/Scripts/Level/Shop/View/ViewImpl.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Sirenix.OdinInspector;
+using UnityEngine;
+
+namespace Level.Shop.View
+{
+    [RequireComponent(typeof(Animator))]
+    [AddComponentMenu("Scripts/Level/Shop/Level.Shop.View")]
+    public partial class ViewImpl : MonoBehaviour
```

Comment:
Make `RoomsView` and `EmployeesView` separate `Monobehaviour`s instead of introducing `partial` class